src/pages/Puray/main.ts

```diff
@@ -0,0 +1,76 @@
+import { pageInterface } from '../pageInterface';
+
+export const Puray: pageInterface = {
+  name: 'Puray',
+  domain: 'https://puray.moe',
+  languages: ['Portuguese'],
+  type: 'anime',
+  isSyncPage(url) {
+    return url.split('/')[3] === 'watch' && !Number.isNaN(Puray.sync.getEpisode(url));
+  },
+  isOverviewPage(url) {
+    return Boolean(url.split('/')[3] === 'anime');
+  },
+  sync: {
+    getTitle(url) {
+      return j
+        .$('span.text-sm')
+        .first()
+        .text()
+        .replace(/\(.*?\)/g, '')
+        .trim();
+    },
+    getIdentifier(url) {
+      return Puray.sync.getTitle(url);
+    },
+    getOverviewUrl(url) {
+      return j.$('span.text-sm').first().parent().prop('href');
+    },
+    getEpisode(url) {
+      return Number(j.$('span.text-lg').first().text().split('-')[0]);
+    },
+  },
+  overview: {
+    getTitle(url) {
+      return j
+        .$('div.text-3xl')
+        .first()
+        .text()
+        .replace(/\(.*?\)/g, '')
+        .trim();
+    },
+    getIdentifier(url) {
+      return Puray.overview!.getTitle(url);
```

Comment:
Returned identifier is not the same as on the video page. As there are multiple seasons on the overview page you would need to check what accordion is open and listening for clicks etc. I recommend to completly remove the overview implementation as it is not really feasable.